SConstruct

```diff
@@ -35,18 +35,23 @@ if env['platform'] == 'x11':
     env.Append(RPATH=env.Literal('\$$ORIGIN'))
     # statically link glibc
     env.Append(LIBS=[File('/usr/lib/x86_64-linux-gnu/libc_nonshared.a')])
+if env['platform'] == 'x11_32':
+    env.Append(RPATH=env.Literal('\$$ORIGIN'))
+    # statically link glibc
+    env.Append(LIBS=[File('/usr/lib/i386-linux-gnu/libc_nonshared.a')])
 
 env.Append(CPPPATH=['#' + include_path + '/'])
 env.Append(CPPPATH=['#godot_include'])
 
 tool_prefix = ''
 if os.name == 'posix' and env['platform'] == 'win64':
     tool_prefix = "x86_64-w64-mingw32-"
-    if (os.getenv("MINGW64_PREFIX")):
-        tool_prefix = os.getenv("MINGW64_PREFIX")
     env['SHLIBSUFFIX'] = '.dll'
     env.Append(CPPDEFINES='WIN32')
-
+if os.name == 'posix' and env['platform'] == 'win32':
+    tool_prefix = "i686-w64-mingw32-"
```

Comment:
I think this should be w32 not w64?

Reply:
I think this should be w32 not w64?

Reply:
erm... On my pc there's installed an `i686-w64-mingw32` package so i wrote `i686-w64-mingw32-` there in `tool_prefix`. I also didn't find an linux package named: `i686-w32-mingw32`. Also `i686-w64-mingw32` works with compilation for 32bit platforms for me so i don't think that there's need to change `i686-w64-mingw32` to `i686-w32-mingw32`

Reply:
Hmm, confusing